src/common/metrics/contact_tracing.ts

```diff
@@ -68,8 +68,10 @@ export const CONTACT_TRACING_LEVEL_INFO_MAP: LevelInfoMap = {
 export function contactTracingStatusText(projection: Projection) {
   const currentContactTracers = projection.currentContactTracers;
   const currentWeeklyAverage =
-    projection.getWeeklyAverageCaseForDay() &&
-    Math.round(projection.getWeeklyAverageCaseForDay() || 0);
+    projection.getWeeklyAverageForDay(projection.dailyPositiveTests) &&
+    Math.round(
+      projection.getWeeklyAverageForDay(projection.dailyPositiveTests) || 0,
+    );
```

Comment:
FWIW (feel free to leave as-is since this is existing code)- this could be cleaned up to something like:
```
  const weeklyAverageResult = projection.getWeeklyAverageForDay(projection.dailyPositiveTests);
  const currentWeeklyAverage = weeklyAverageResult && Math.round(weeklyAverageResult);
```

It reduces duplication and avoids `|| 0` since if you use an intermediate variable, TypeScript knows that inside the Math.round(), `weeklyAverageResult` is guaranteed to be a number (not null).